datasets/cl/info_probidad/crawler.py

```diff
@@ -0,0 +1,48 @@
+from pantomime.types import CSV
+from urllib.parse import urlparse
+from typing import Dict
+import os
+import csv
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    person = context.make("Person")
+
+    id = os.path.basename(urlparse(row.pop("UriDeclarante")).path)
+    person.id = context.make_slug(id)
+
+    h.apply_name(
+        person,
+        first_name=row.pop("Nombre"),
+        patronymic=row.pop("ApPaterno"),
+        matronymic=row.pop("ApMaterno"),
+    )
+
+    position_name = row.pop("Cargo")
+    position = h.make_position(
+        context, position_name, country="cl", lang="spa"
+    )
+    position_lookup = context.lookup("positions", position_name)
+    
+    # all positions (ie. Cargo) should be explicitly classified as either pep or no-pep
+    if not position_lookup:
+        context.audit_data(position.to_dict())
```

Comment:
We usually do audit_data on input data, e.g. a dict representing a row in a table. So at the end of the crawler, you might want to do context.audit_data(row) to see if there are fields left that you could well use, otherwise add them to the `ignroe` argument to `audit_data`.

Reply:
Do you think it makes sense to log as warnings if new values for `Cargo` are identified? I was doing it before because I though that the categorisation was going to happen in `datasets/cl/info_probidad/cl_info_probidad.yml`, but now I'm not so sure because the positions will end un the database anyway.

Reply:
Yeah not a bad idea! I like how this is working now - so the list tells us whether we've taken a look at the role before, and the database tells us whether it's a PEP or not.

Once this is running for a few months we can see whether this changes often, if ever.